frontend/angular.json

```diff
@@ -48,7 +48,10 @@
                         "assets": ["src/favicon.ico", "src/assets"],
                         "styles": ["src/styles/index.sass"],
                         "scripts": [],
-                        "allowedCommonJsDependencies": ["moment-timezone"]
+                        "allowedCommonJsDependencies": ["moment-timezone"],
+                        "stylePreprocessorOptions": {
+                            "includePaths": ["src/styles"]
+                          }
```

Comment:
aliases you be here 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/38536017/7d41cc55-10fd-4a61-828d-220cbb10d18f)

Reply:
From what I discovered, sass file can't use typescript aliases, am I right?

Reply:
yes, you are right, sorry
your solution is valid